### PR TITLE
Fixup link to Rntime

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/migration/22-to-30.md [hunk 1]
```diff
@@ -525,4 +525,4 @@ If you're using features of `Newtonsoft.Json` that aren't supported in `System.T
 
 ## Opt in to runtime compilation
 
-In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see <xref:mvc/views/view-compilation#runtime-compilation>.
+In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation.
```
Comment (pranavkm) [inline aspnetcore/migration/22-to-30.md]: Is there a better way to do this? Right now the migration notes take you to the 2.2 docs which aren't helpful.

Comment (scottaddie) [inline aspnetcore/migration/22-to-30.md]:
```suggestion
In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see https://docs.microsoft.com/aspnet/core/mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation.
```

Comment (scottaddie) [reply aspnetcore/migration/22-to-30.md]: This is how I would've handled it.

**Comment by scottaddie** [comment]
Thank you, @pranavkm!

**Pull request merged**